[WIP] Meine Änderungen...

r0x0r316 opened the pull request:
Bin bislang nicht dazu gekommen das ganze etwas aufzuräumen, hab in der readme nen paar Änderungen genannt. Wichtigste Änderungen:
- PHP 7 kompatibel 
- Composer für das updaten der libs
- Passwordleerzeichen fix
- Normalisierung der Releasenamen 
- Verbesserung der Suche und mehr Treffer
- Suche per ids (jede menge fremdklassen imdb,tvrage,tvdb usw.
- UHD Kategorien
Weiss gar nicht mehr was ich noch alles geändert habe ich glaube das beschreibt aber das meiste.
Ich hoffe ich hab nicht allzu viele Fehler gemacht und die leute bekommens auf Anhieb zum laufen.
Das läuft bei mir nun seit ca. 6 monaten in der Konfiguration und hab seitdem nicht einen passwort fehler oder sonstiges. Ist kompatibel mit radarr sonarr und nzbhydra (wichtig nur eine ID für die suche angeben in den einstellungen werden alle erkannt)
Bis die Tage ich komm wieder wenn ich das ganze finalisiert habe.

Commit:
Add files via upload

Commit:
Delete index.php

Commit:
This is a general overhaul of cytecs script updated it to be php7 compatible and also made changes to the searches more can be found in the readme
	geändert:       README.md
	geändert:       docker-compose.yml
	geändert:       docker/Dockerfile
	geändert:       httpdocs/admin/.htaccess
	neue Datei:     httpdocs/inc/TVMaze/AKA.php
	neue Datei:     httpdocs/inc/TVMaze/Actor.php
	neue Datei:     httpdocs/inc/TVMaze/Character.php
	neue Datei:     httpdocs/inc/TVMaze/Client.php
	neue Datei:     httpdocs/inc/TVMaze/Crew.php
	neue Datei:     httpdocs/inc/TVMaze/Episode.php
	neue Datei:     httpdocs/inc/TVMaze/TVMaze.php
	neue Datei:     httpdocs/inc/TVMaze/TVProduction.php
	neue Datei:     httpdocs/inc/TVMaze/TVShow.php
	neue Datei:     httpdocs/inc/composer.json
	geändert:       httpdocs/inc/config.inc.php.sample
	neue Datei:     httpdocs/inc/mcrypt.php
	geändert:       httpdocs/inc/nzbto.php
	geändert:       httpdocs/inc/simple_html_dom.php
	neue Datei:     httpdocs/inc/vendor/autoload.php
	neue Datei:     httpdocs/inc/vendor/composer/ClassLoader.php
	neue Datei:     httpdocs/inc/vendor/composer/LICENSE
	neue Datei:     httpdocs/inc/vendor/composer/autoload_classmap.php
	neue Datei:     httpdocs/inc/vendor/composer/autoload_files.php
	neue Datei:     httpdocs/inc/vendor/composer/autoload_namespaces.php
	neue Datei:     httpdocs/inc/vendor/composer/autoload_psr4.php
	neue Datei:     httpdocs/inc/vendor/composer/autoload_real.php
	neue Datei:     httpdocs/inc/vendor/composer/autoload_static.php
	neue Datei:     httpdocs/inc/vendor/composer/installed.json
	neue Datei:     httpdocs/inc/vendor/fabianbeiner/php-imdb-grabber/.editorconf
	neue Datei:     httpdocs/inc/vendor/fabianbeiner/php-imdb-grabber/.gitignore
	neue Datei:     httpdocs/inc/vendor/fabianbeiner/php-imdb-grabber/CHANGELOG.md
	neue Datei:     httpdocs/inc/vendor/fabianbeiner/php-imdb-grabber/LICENSE
	neue Datei:     httpdocs/inc/vendor/fabianbeiner/php-imdb-grabber/README.md
	neue Datei:     httpdocs/inc/vendor/fabianbeiner/php-imdb-grabber/cache/.gitkeep
	neue Datei:     httpdocs/inc/vendor/fabianbeiner/php-imdb-grabber/cast/not-found.jpg
	neue Datei:     httpdocs/inc/vendor/fabianbeiner/php-imdb-grabber/composer.json
	neue Datei:     httpdocs/inc/vendor/fabianbeiner/php-imdb-grabber/examples/imdb.example.php
	neue Datei:     httpdocs/inc/vendor/fabianbeiner/php-imdb-grabber/examples/imdb.tests.php
	neue Datei:     httpdocs/inc/vendor/fabianbeiner/php-imdb-grabber/imdb.class.php
	neue Datei:     httpdocs/inc/vendor/fabianbeiner/php-imdb-grabber/posters/not-found.jpg
	neue Datei:     httpdocs/inc/vendor/joshpinkney/tv-maze-php-api/Examples.php
	neue Datei:     httpdocs/inc/vendor/joshpinkney/tv-maze-php-api/LICENSE.md
	neue Datei:     httpdocs/inc/vendor/joshpinkney/tv-maze-php-api/README.md
	neue Datei:     httpdocs/inc/vendor/joshpinkney/tv-maze-php-api/TVMaze/AKA.php
	neue Datei:     httpdocs/inc/vendor/joshpinkney/tv-maze-php-api/TVMaze/Actor.php
	neue Datei:     httpdocs/inc/vendor/joshpinkney/tv-maze-php-api/TVMaze/Character.php
	neue Datei:     httpdocs/inc/vendor/joshpinkney/tv-maze-php-api/TVMaze/Client.php
	neue Datei:     httpdocs/inc/vendor/joshpinkney/tv-maze-php-api/TVMaze/Crew.php
	neue Datei:     httpdocs/inc/vendor/joshpinkney/tv-maze-php-api/TVMaze/Episode.php
	neue Datei:     httpdocs/inc/vendor/joshpinkney/tv-maze-php-api/TVMaze/TVMaze.php
	neue Datei:     httpdocs/inc/vendor/joshpinkney/tv-maze-php-api/TVMaze/TVProduction.php
	neue Datei:     httpdocs/inc/vendor/joshpinkney/tv-maze-php-api/TVMaze/TVShow.php
	neue Datei:     httpdocs/inc/vendor/joshpinkney/tv-maze-php-api/TVMazeIncludes.php
	neue Datei:     httpdocs/inc/vendor/joshpinkney/tv-maze-php-api/composer.json
	neue Datei:     httpdocs/inc/vendor/phpseclib/mcrypt_compat/.styleci.yml
	neue Datei:     httpdocs/inc/vendor/phpseclib/mcrypt_compat/AUTHORS
	neue Datei:     httpdocs/inc/vendor/phpseclib/mcrypt_compat/LICENSE
	neue Datei:     httpdocs/inc/vendor/phpseclib/mcrypt_compat/README.md
	neue Datei:     httpdocs/inc/vendor/phpseclib/mcrypt_compat/composer.json
	neue Datei:     httpdocs/inc/vendor/phpseclib/mcrypt_compat/lib/mcrypt.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/AUTHORS
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/LICENSE
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/README.md
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/appveyor.yml
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/composer.json
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/Crypt/AES.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/Crypt/Base.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/Crypt/Blowfish.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/Crypt/DES.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/Crypt/Hash.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/Crypt/RC2.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/Crypt/RC4.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/Crypt/RSA.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/Crypt/Random.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/Crypt/Rijndael.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/Crypt/TripleDES.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/Crypt/Twofish.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/File/ANSI.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/File/ASN1.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/File/ASN1/Element.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/File/X509.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/Math/BigInteger.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/Net/SCP.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/Net/SFTP.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/Net/SFTP/Stream.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/Net/SSH1.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/Net/SSH2.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/System/SSH/Agent.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/System/SSH/Agent/Identity.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/bootstrap.php
	neue Datei:     httpdocs/inc/vendor/phpseclib/phpseclib/phpseclib/openssl.cnf
	geändert:       httpdocs/index.php
	geändert:       httpdocs/libs/simple_html_dom.php
	geändert:       httpdocs/libs/smarty/configs/smarty.conf
	geändert:       httpdocs/libs/smarty/debug.tpl
	geändert:       httpdocs/theme/templates/apicaps.tpl
	geändert:       httpdocs/theme/templates/apiresult.tpl
	geändert:       httpdocs/theme/templates/basepage.tpl
	geändert:       httpdocs/theme/templates/index.tpl

Commit:
revert port changes in compose
	geändert:       README.md
	geändert:       docker-compose.yml

Commit:
Update nzbto.php

Commit:
remove comment
	geändert:       httpdocs/index.php

Commit:
Merge branch 'master' of https://github.com/r0x0r316/nzb.to-nZEDb

Commit:
regex fix
	geändert:       httpdocs/index.php

cytec commented:
Ich hab das jetzt erst mal nur grob durchgescrollt aber das schaut soweit gut aus. Ich änder den Titel mal auf WIP (wenn ich dich richtig verstanden habe ist das noch nicht 100% final?)

EDIT: ich hab nur schnell nen quick test gemacht (pull und docker-compose) läuft auf jeden fall alles, ausgiebig testen kann ich das ganze leider nicht mehr da ich kein Acc mehr habe, da vertrau ich dir einfach mal... 

Mal sehen, vll findet sich ja noch jemand der das testen will \cc @wizzardking @BMWfan

ghost commented:
Freut mich :D, Ja aber denke soweit sind die meisten änderungen schon so weit dass man das übernehmen kann wie gesagt läuft seit sechs monaten bei und hatte nen die ersten drei monate nen heftigen 100 tb backfill :P also da sind die meisten fehler ausgemerzt worden. kommen aber wie gesagt noch audiosuche und multi id suche dazu so wie ich etwas zeit dafür finde ;)

Commit:
Massive fixes. Audiosearch support, Lidarr Support, Audiobooks support
TV-Regex fixes, got rid of some loops. Cleanup.
Multicat now fully working if searchtool request 2045,2050,etc each cat will deliver 50 results.
New Category indexes:
id="2045" name="UHD-Filme
id="2050" name="3D"
id="2060" name="BluRay"
id="2070" name="X265"
id="2080" name="MP4"

id="3000" name="Audio"
id="3010" name="MP3"
id="3030" name="Audiobooks"
id="3040" name="FLAC"

id="5030" name="Series"
id="5045" name="UHD-Series"
id="5080" name="Dokus"/>
	geändert:       inc/nzbto.php
	geändert:       index.php
	geändert:       theme/templates/apicaps.tpl

Commit:
Lidarr audio search fix.TODO regex filter for audio rlses.
	geändert:       index.php

Commit:
Tv Regex now finds special cases like Magnum P.I. series with german subtitles like Ancient.Aliens-Unerklaerliche... and so on.
Fixed apicaps to reflect changes of search-abilities
	geändert:       index.php
	geändert:       theme/templates/apicaps.tpl

Commit:
fix multicat caching and provide disinction of category in log
	geändert:       inc/nzbto.php
	geändert:       index.php
	geändert:       theme/templates/apicaps.tpl

3OW committed:
Update Dockerfile

Commit:
Merge pull request #1 from 3OW/master

Update Dockerfile
